Reshuffle documentation (#16688)
strapi/strapi: docs/docs/index.md

---

# Strapi contributor documentation

Welcome to the Strapi Contributor documentation.

This documentation site is a constant WIP so please, continue to add and improve these docs. The general layout focusses on 4 key areas:

## Guides

> This is where you'll probably want to start off. We have our contributing guides & code of conduct which are very important to read. There's also useful guides on common situations whilst developing such as ["Working with the Design System"](/guides/working-with-the-design-system). Or more high level guides such as best practices for frontend development (coming soon).

Reviewer: ```suggestion
This is where you'll probably want to start off. We have our contributing guides & code of conduct which are very important to read. There are also useful guides on common situations whilst developing such as ["Working with the Design System"](/guides/working-with-the-design-system) and higher-level guides such as best practices for frontend development (coming soon).
```

## Docs

> Within the docs section we have a multitude of both technical and conceptual documentation diving deep into particular parts of the Strapi monorepo that may not make as much sense just reading the code like ["Relations reordering in the database"](/docs/core/database/relations/reordering). There's also usage documentation for various pieces of code such as the [useDragAndDrop](/docs/core/content-manager/hooks/use-drag-and-drop) hook.

Reviewer: ```suggestion
Within the docs section we have a multitude of both technical and conceptual documentation diving deep into particular parts of the Strapi monorepo that may not make as much sense as just reading the code, like ["Relations reordering in the database"](/docs/core/database/relations/reordering). There's also usage documentation for various pieces of code such as the [useDragAndDrop](/docs/core/content-manager/hooks/use-drag-and-drop) hook.
```

## API Reference

An advanced deep dive into some of the core driving classes of Strapi with explanations on the methods & parameters available on commonly exposed classes as well as examples to compliment them for easier understanding.

## RFCs

> A growing section we intend to populate over time with public facing RFCs once approve as a record. These assist in understanding the design direction of features and code to understand the contextual "whys" that may not be apparent.

Reviewer: ```suggestion
A growing section we intend to populate over time with public-facing RFCs once approved to maintain as a record. These assist in understanding the design direction of features and code to understand the contextual "whys" that may not be apparent.
```
